docs/additional-resources/articles.table.js

```diff
@@ -68,7 +68,7 @@ export const articles = [
     title: "Writing Pester Tests",
     author: "PowerShell Community",
     date: "2016-11-07",
-    url: "https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md",
+    url: "https://github.com/PowerShell/PowerShell/blob/main/docs/testing-guidelines/WritingPesterTests.md",
```

Comment:
```suggestion
    url: "https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md",
```

They still use master 🙂

Reply:
Ohw, totally missed that and totally missed that `Commit Suggestion` button as well. Never seen it before but that's amazing. Thanks 👍

Reply:
No worries. Lightning quick response as usual 😄